chat-widget/src/components/livechatwidget/common/ActivitySubscriber/BotAuthActivitySubscriber.ts

```diff
@@ -86,7 +86,10 @@ export class BotAuthActivitySubscriber implements IActivitySubscriber {
         const signInId = extractSignInId(signInUrl);
 
         if (!signInId) {
-            return;
+            TelemetryHelper.logLoadingEvent(LogLevel.INFO, {
+                Event: TelemetryEvent.BotAuthActivityUndefinedSignInId
```

Comment:
Please update telemetry.md with this change

Reply:
Done